### PR TITLE
DOCO-871 Update Migration Guide

### LINKED ISSUE #8160: Upgrade a Site That Was Created With an Empty Upstream to the Latest Version of Drupal Doc Update
Re: [Upgrade a Site That Was Created With an Empty Upstream to the Latest Version of Drupal](https://docs.pantheon.io//guides/drupal-hosted-createempty-md/composer-upstream)

Priority: Medium

## Issue Description:

The `composer-upstream` page has a few problems.

1. It includes several commands like `git checkout main config` even though the default branch is normally (always?) `master` and not `main`.
3. If the existing site uses a nested document root (such as `web/`) then the command `git mv config/* config` is redundant.
4. In either case, `git rm -f web/sites/default/config/.htaccess` should refer to where the file is now: `git rm -f config/.htaccess`.

It seems a little odd to copy configuration before installing modules. I guess those steps can be done in either order, as long as both are done before we try to run the site (and import configuration).

## Suggested Resolution

### PR DESCRIPTION
**Pull request by wordsmither**
Closes #8160

## Summary

**[Upgrade a Site That Was Created With an Empty Upstream to the Latest Version of Drupal](https://docs.pantheon.io//guides/drupal-hosted-createempty-md/composer-upstream)** - Remove redundant command. Don't delete config/.htaccess files

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by wordsmither** [comment]
[Preview](https://pr-8162-documentation.appa.pantheon.site/guides/drupal-hosted-createempty-md/composer-upstream)

File: source/content/guides/drupal/drupal-hosted-createempty-md/05-composer-upstream.md [hunk 1]
```diff
@@ -74,8 +74,6 @@ This is a common location for the `config` file. If this isn't where your `confi
 
 ```bash{promptUser:user}
 git checkout main config
-git mv config/* config
-git rm -f web/sites/default/config/.htaccess
```
Comment (benjifisher) [inline source/content/guides/drupal/drupal-hosted-createempty-md/05-composer-upstream.md:78]: Do we want to remove Line 78 or replace it with

```markdown
git rm -f config/.htaccess
```

Comment (greg-1-anderson) [reply source/content/guides/drupal/drupal-hosted-createempty-md/05-composer-upstream.md:78]: .htaccess doesn't affect Pantheon behavior. Some sniffers check to see if it's there and complain if it isn't. Might as well leave it there.

**Review by michellecolon-pantheon — APPROVED** [review]
LGTM

**Pull request merged**